### ISSUE TITLE
Conflict with junegunn/gv.vim, Buffer 50 does not exist

### ISSUE THREAD

**Issue by PSeitz** [kyazdani42/nvim-tree.lua#366]
When I execute `:GV` from https://github.com/junegunn/gv.vim/ I get following nvim tree error:

Error executing vim.schedule lua callback: ...im/autoload/plugged/nvim-tree.lua/lua/nvim-tree/view.lua:162: Vim(buffer):E86: Buffer 50 does not exist

**Comment by kyazdani42** [comment]
hum i guess gv.vim overwrites the current buffer which is nvim-tree buffer (or kills it) which is weird. I'd have to dig into gv.vim to understand how it loads its buffers, because this error relates the buffer_override prevention which i find strange.

**Comment by Iron-E** [comment]
I'm running into this with `neogit` as well. Every time I run `:Neogit` I get the following error:

```log
Error executing vim.schedule lua callback: ...e/pack/packer/start/nvim-tree.lua/lua/nvim-tree/view.lua:191: Vim(buffer):E86: Buffer 9 does not exist
```

The number '9' is inconsequential— it increases by one every time I invoke `:Neogit`.

**Comment by kyazdani42** [comment]
should be fixed in 44678adc1e6ab

**Comment by Iron-E** [comment]
Can confirm!
